packages/core/src/alert/alert.element.spec.ts

```diff
@@ -229,7 +213,7 @@ describe('Alert element – ', () => {
       await componentIsStable(component);
       const slots = getComponentSlotContent(component);
       expect(slots.actions.trim()).toBe(
-        `<cds-alert-actions slot="actions" type="default">${placeholderActionsText}</cds-alert-actions>`
+        `<cds-alert-actions slot="actions" type="default"><!---->${placeholderActionsText}<!----></cds-alert-actions>`
```

Comment:
lit-html `render` adds some markers for white space which was slightly different than the innerHTML output prior